### PR TITLE
Removing Apply Fix success notification removed.

### PR DESCRIPTION
**Pull request by ashishrp-aws**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Description

- Added Listener for triggering auto file scan for opening files in editor.
- Removed User Notifications in case of file scans, apply fix success
- Removing logs for files scans. 
- changing payload collection method for effective traversal. 
- Fix for Stuck stop button by ensuring quick job cancellation. 
- Removed redundant tests
- Fix for programming language for payloadContext of project scan.Instead of sending Unknown/Plaintext, sending highest occurring language in payloadContext.

## Checklist
- [x] My code follows the code style of this project
- [x] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by ctlai95 — APPROVED** [review]

**Review by ctlai95 — APPROVED** [review]

File: plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanSession.kt [hunk 1]
```diff
@@ -263,10 +279,21 @@ class CodeWhispererCodeScanSession(val sessionContext: CodeScanSessionContext) {
...
-    fun uploadArtifactToS3(url: String, fileToUpload: File, requestHeaders: Map<String, String>?) {
+    fun uploadArtifactToS3(url: String, uploadId: String, fileToUpload: File, md5: String, kmsArn: String?, requestHeaders: Map<String, String>?) {
+        val uploadIdJson = """{"uploadId":"$uploadId"}"""
         HttpRequests.put(url, "application/zip").userAgent(AwsClientManager.getUserAgent()).tuner {
-            requestHeaders?.forEach { entry ->
-                it.setRequestProperty(entry.key, entry.value)
+            if (requestHeaders.isNullOrEmpty()) {
+                it.setRequestProperty(CONTENT_MD5, md5)
+                it.setRequestProperty(CONTENT_TYPE, APPLICATION_ZIP)
+                it.setRequestProperty(SERVER_SIDE_ENCRYPTION, AWS_KMS)
```
Comment (manodnyab) [inline plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanSession.kt:288]: why is this required?

Comment (ashishrp-aws) [reply plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanSession.kt:288]: This is fallback if request headers are empty. the new QCA call takes request headers and old RTS prod takes the old info

**Review by ctlai95 — APPROVED** [review]

**Review by manodnyab — APPROVED** [review]
